prow/cluster/jobs/rh-ibm-synergy/multicloud-operators-foundation/IBM.multicloud-operators-foundation.master.yaml

```diff
@@ -0,0 +1,87 @@
+presubmits:
+  rh-ibm-synergy/multicloud-operators-foundation:
+  - name: build_multicloud-operators-foundation
+    branches:
+    - ^master$
+    decorate: true
+    always_run: true
+    path_alias: github.ibm.com/IBMPrivateCloud/multicloud-operators-foundation
+    rerun_command: /test build multicloud-operators-foundation
+    spec:
+      containers:
+      - command:
+        - make
+        - build
+        image: quay.io/multicloudlab/build-tools:2019-10-08T07-03-25
+        name: ""
+        securityContext:
+          privileged: true
+    trigger: '(?m)^/test (?:.*? )?build(?: .*?)?$'
+  - name: test_multicloud-operators-foundation
+    branches:
+    - ^master$
+    decorate: true
+    always_run: true
+    path_alias: github.ibm.com/IBMPrivateCloud/multicloud-operators-foundation
+    rerun_command: /test test multicloud-operators-foundation
+    spec:
+      containers:
+      - command:
+        - make
+        - test
+        image: quay.io/multicloudlab/build-tools:2019-10-08T07-03-25
+        name: ""
+        securityContext:
+          privileged: true
+    trigger: '(?m)^/test (?:.*? )?test(?: .*?)?$'
+
+postsubmits:
+  IBM/multicloud-operators-foundation:
+  - name: test_multicloud-operators-foundation_postsubmit
+    branches:
+    - ^master$
+    decorate: true
+    path_alias: github.ibm.com/IBMPrivateCloud/multicloud-operators-foundation
+    spec:
+      containers:
+      - command:
+        - make
+        - test
+        image: quay.io/multicloudlab/build-tools:2019-10-08T07-03-25
+        name: ""
+        securityContext:
+          privileged: true
+  - name: coverage_multicloud-operators-foundation_postsubmit
+    branches:
+    - ^master$
+    decorate: true
+    path_alias: github.ibm.com/IBMPrivateCloud/multicloud-operators-foundation
+    spec:
+      containers:
+      - command:
+        - make
+        - coverage
+        image: quay.io/multicloudlab/build-tools:2019-10-08T07-03-25
+        name: ""
+        env:
+        - name: CODECOV_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: ibm-multicloud-operators-foundation-codecov-token
+              key: codecov-token
```

Comment:
To enable CODECOV, you need to authorize https://codecov.io/ to access your repo and generate code coverage upload token, and then I can help you to create CODECOV_TOKEN  secret, ping me in slack channel.

Reply:
I don't think we want to enable CODECOV at this stage, the codecov report contains file diff and disclosues code.  We shall enable this after it is fully public.

Reply:
Correct, let's remove this section for now.

Reply:
done